tests/testthat/rmds/fig-size-knitr-options.Rmd

```diff
@@ -0,0 +1,42 @@
+---
+title: Test Fig Size Knitr Options
+output: flexdashboard::flex_dashboard
+---
+
+<!-- TEST -->
+
+## Column 1
+
+### A wide plot
+
+```{r, fig.width=10, fig.height=5}
+hist(faithful$waiting)
+```
+
+### A tall plot
+
+```{r, fig.width=5, fig.height=10}
+hist(faithful$waiting)
+```
+
+## Column 2
+
+### 75 DPI, Retina 1
+
+```{r, fig.width=5, fig.height=5, dpi=75, fig.retina=1}
+hist(faithful$waiting)
+```
+
+### 150 DPI, Retina 1
+
+```{r, fig.width=5, fig.height=5, dpi=150, fig.retina=1}
+hist(faithful$waiting)
+```
+
+### 75 DPI, Retina 2
+
+```{r, fig.width=5, fig.height=5, dpi=75, fig.retina=2}
+hist(faithful$waiting)
+```
```

Comment:
IIUC, these dpi/retina options don't influence the snapshot result (in a way that we actually care about), so let's remove them? 

```suggestion
```

Reply:
Yeah, you're right, they don't affect our snapshot. I was on the fence about them but I included them in the PR because they'd alert anyone attempt a refactor in the future that we currently don't account for DPI in the conversion to pixels. But on the whole I agree that it's probably better to take them out.

Reply:
Done in 763ff60c24cfcb64f98a09da27d755fecb04c2e7